<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AURA 관제 대시보드</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="main_layout" stretch="7,3">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QWidget" name="video_widget_placeholder" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="video_container_layout"/>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="control_panel" native="true">
      <layout class="QVBoxLayout" name="control_layout">
       <item>
        <widget class="QPushButton" name="log_open_button">
         <property name="text">
          <string>로그 뷰어 및 동영상 재생</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="alert_groupbox">
         <property name="title">
          <string>실시간 알림 목록</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QListWidget" name="alert_list_widget"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_resolve_alert">
            <property name="text">
             <string>선택 알림 확인 (종료)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>통합 이벤트 로그</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="log_browser"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
